src/test/rgw/CMakeLists.txt

```diff
@@ -388,21 +388,19 @@ target_link_libraries(ceph_test_datalog
   ${rgw_libs})
 install(TARGETS ceph_test_datalog DESTINATION ${CMAKE_INSTALL_BINDIR})
 
-if(WITH_CATCH2)
- add_executable(unittest_rgw_hex
-  test_rgw_hex.cc)
+#
+# Helper for adding RGW tests built with Catch2:
+# Note: to define a test that does not use Catch2::main(), use:
+#   add_catch2_test_rgw(my_test_name NO_CATCH2_MAIN)
+# ...your test should be in a file called my_test_name.cc and the output will be
```

Comment:
It should actually be "test_<my-test-name>.cc" (ex. "test_foo.cc"). Will fix comment.